api/types/types.proto

```diff
@@ -2624,6 +2634,90 @@ message GithubConnectorSpecV3 {
     string Display = 5 [ (gogoproto.jsontag) = "display" ];
 }
 
+// SAMLAttributesToRolesWarnings contains warnings produced during the process of mapping the SAML
+// attributes to roles.
+message SAMLAttributesToRolesWarnings {
```

Comment:
Does this type need to have such a specific name? Can it be named something like `SSOTestWarnings` and reused for other connector flows too?

Reply:
I don't know if there will be other users for such a type, but indeed we can pick a more generic name here.